Public/Src/Cache/ContentStore/DistributedTest/TestErrorResultConverter.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using BuildXL.Cache.ContentStore.Interfaces.Results;
+using Xunit;
+
+namespace BuildXL.Cache.ContentStore.Distributed.Test
+{
+    public class TestErrorResultConverter
+    {
+        /// <summary>
+        /// Validate that all implementations of ResultBase include the necessary constructor to use the AsResult extension method.
+        /// </summary>
+        [Fact]
+        public void ValidateAsResult()

```

Comment:
Nit, should this test case be in ErrorResultConverterTests?
Minor, but it maybe easier to have one test suite instead of more then one.

Reply:
ErrorResultConverterTests is in BuildXL.Cache.ContentStore.Interfaces.Test, which does not include our distributed bits. To validate this feature, we need to cover more ground.